tests/acceptance/25_cf-execd/dies_in_time.cf

```diff
@@ -0,0 +1,155 @@
+# ENT-3147, zd#3157
+
+# Ensure that cf-execd dies within 3 seconds after being signalled.
```

Comment:
Yes, leaving processes running has been challenging in the timed tests. I think it's alright to leave it as a standard test.

No strong opinion on the `slow` category, but I'm wondering when this would be useful. In a parallel execution, and considering the other, near 2000 tests, I don't see it as a big deal, it will hardly be noticable. In the CI we would always need to run it regardless.